RELEASING.md

```diff
@@ -0,0 +1,27 @@
+# Releasing
+
+1. Update the version number in these places:
+  - `lib/neat/version.rb`
+  - `core/neat/_neat.scss`
+  - `package.json`
+  - `bower.json`
+
+1. Update `NEWS.md`
+
+1. Commit changes. Use the convention “Neat vX.X.X” in your commit message.
+   There shouldn’t be code changes, and thus CI doesn’t need to run.
+
+1. Run `rake release`, which tags the release, pushes the tag to GitHub, and
```

Comment:
@wardpenney Yeah, this is weird, but it’s part of Bundler, and since we `require "bundler"` in our Rakefile, it’s available to us.